vassal-app/src/main/java/VASSAL/build/module/GameState.java

```diff
@@ -332,15 +345,15 @@ public void setup(boolean gameStarting) {
     closeGame.setEnabled(gameStarting);
 
     if (gameStarting) {
-      loadGame.putValue(Action.NAME,
-        Resources.getString("GameState.load_continuation"));
+      //loadGame.putValue(Action.NAME, Resources.getString("GameState.load_continuation"));
       g.getWizardSupport().showGameSetupWizard();
     }
     else {
-      loadGame.putValue(Action.NAME,
-        Resources.getString("GameState.load_game"));
+      //loadGame.putValue(Action.NAME, Resources.getString("GameState.load_game"));
     }
```

Comment:
Remove the empty else. (Probably no reason to keep the commented line in the if either.)

Reply:
Oh right, this mess brought to you by the "let's see if this will work at all" phase